### ISSUE TITLE
CentOS 7.7 1908 support

### ISSUE THREAD

**Issue by JonKohler** [CentOS/sig-cloud-instance-images#150]
Is this the right place to track requests for 7.7 1908 support as a docker image? or is that another repo?

**Comment by chewi** [comment]
I'm also interested in the time frame for this. I believe it appeared after a couple of days last time. Is there something holding it up? Is there anything we can do to help?

**Comment by jperrin** [comment]
This will get added in the monthly image run, in just a few days.

**Comment by olifre** [comment]
Will the monthly run coincidentally also add CentOS 8, or should a different issue be opened for that (it's out only for a few hours, but asking does not hurt :wink: ))?

**Comment by chewi** [comment]
I see [a new build](https://ci.centos.org/job/sig-cloudinstance-docker-baseimages-bootstrap/152/) has happened, assuming this is the right one. I thought the rest would happen automatically but now I see you take that tarball and commit it here.

**Comment by chewi** [comment]
Some movement here would be appreciated. Sorry to sound impatient but I'm under some pressure to deliver something based on this. I would just build it myself but we need our images to be based on something official. I'd also file a pull request but it looks like you literally just commit the files that were generated from CI.

**Comment by chewi** [comment]
I gather from the centos-devel list that there was concern over the size of the 8 image but it ended up being published anyway. It's not clear whether that applied to the 7.7 image too.

**Comment by TommiFore** [comment]
I'm also under similar pressure and it would be great to have 7.7 but anyway, thanks for great effort.

**Comment by matttbe** [comment]
> I'm also under similar pressure and it would be great to have 7.7 but anyway, thanks for great effort.

As a workaround, can you not create a "fake" 7.7 image?

```dockerfile
FROM centos:7.6.1810

RUN yum -y install centos-release 7-7.1908.0.el7.centos && \
    yum -y update && \
    yum -y clean all
# (last step is optional)
```

Then create your own centos image, tag it and push it to your registry.

**Comment by TommiFore** [comment]
> As a workaround, can you not create a "fake" 7.7 image?

Sure, and now I will. There are still dozens of other packages to update too. Building is so much faster when those packages are up-to-date immediately.

**Comment by chewi** [comment]
You'd be better just building from [these files](http://artifacts.ci.centos.org/sig-cloudinstance/centos-7-191001/x86_64/) as that's essentially where the final image will come from.

**Comment by TommiFore** [comment]
Oh, thanks. Thats enough for me because I'm building from scratch with that file anyway. Hope you will get your image also...

**Comment by qhaas** [comment]
Glad this is in the works, handy since python3 has 'graduated' from EPEL to the base repos, therefore package names have changed.

**Comment by JonKohler** [comment]
Bumping this again to see where we're at here? Is this going to fall into next month's run? or is there another schedule we can track to?

As @TommiFore mentioned above, having this pre-baked is very helpful to us, as it minimizes the amount of variables for the typical internal qual churn one might have.

**Comment by michalvanco** [comment]
Any updates here? Or ETA where we can get official 7.7 image on docker hub? Thx!

**Comment by pasikarkkainen** [comment]
@jperrin Should be able to provide an update about the centos 7.7 container image status..

**Comment by pasikarkkainen** [comment]
CentOS 7.7 container images are now on Dockerhub.

**Comment by JonKohler** [comment]
Indeed, done here: https://github.com/CentOS/sig-cloud-instance-images/commit/dcf7932cbda6dd9865d50bfe969927e3e1f0c671

**Comment by igor-raits** [comment]
@JonKohler yes, but the cloud image is still no available. Please reopen.

**Comment by JonKohler** [comment]
Ahh! Gotcha, thanks for the catch @ignatenkobrain

**Comment by igor-raits** [comment]
@jperrin any updates here?

**Comment by davidmonro** [comment]
Still no 7.7 cloud image? I was hoping that would have been done (given 8.1 has finally delivered one for 8!)

**Comment by ImmoWetzel** [comment]
Ping ... any news ? What do I have to do to create this by my own ?

**Comment by nbht** [comment]
I am unable to find centos 7.7 cloud image @https://cloud.centos.org/centos/7/images/ 
Kindly please confirm when it will be availabel

**Comment by smalenfant** [comment]
Same here, cloud-init broken for OpenStack prior to 1908. What's preventing this image from showing up? How can we help?

**Comment by jurajama** [comment]
It seems that Centos-7.8 image has appeared in https://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-2003.qcow2

That really is 7.8 rather than 7.7, I checked by deploying an instance and looking at /etc/centos-release which says "CentOS Linux release 7.8.2003 (Core)"
